src/util/handlebars/handlebar-helpers.ts

```diff
@@ -7,6 +7,7 @@ export const replaceSpaceWithHyphenHelper = (str?: string) => !isPlainObject(str
 
 export const registerHandlebarsHelpers = () => {
 	hbs.registerHelper("toLowerCase", toLowercaseHelper);
+	hbs.registerHelper("or", (...args: unknown[]) => Array.prototype.slice.call(args, 0, -1).some(Boolean));
```

Comment:
I am not following what is the purpose of this one. Maybe write a unit test to demo the purpose?
And how is it different to `[...args].some(Boolean)`? why `-1`?

Reply:
- Helper method re-worked.
- Test cases added for this method and also for `concatStringHelper`, which I had created in the last PR.